src/mmw/js/src/compare/templates/compareScenarios.html

```diff
@@ -0,0 +1,7 @@
+<div id="edge-fade"></div><!-- Blur offcanvas scenario -->
+
+<table> <!-- Table allows for horizontal scrolling. -->
```

Comment:
Horizontal scrolling wasn't working for me, and https://github.com/lewfish/model-my-watershed/commit/684224a9879ee696edd387f8d5588fc1045ad6af makes me think that it shouldn't work. Personally I'd like it to be there, I think it would make it a lot more intuitive on touch screens.